Changelog.md

```diff
@@ -1,7 +1,7 @@
 ### 0.8.29 (unreleased)
 
 Language Features:
- * Introduce syntax for specifying contract storage base location.
+ * Introduce syntax for specifying contract storage layout base.
```

Comment:
Why change it here and not in the base PR?

Reply:
I guess I mixed the PRs :sweat_smile:

Reply:
Actually, I think this is appearing here because this PR is out of sync with the base PR...